client/landing/stepper/index.tsx

```diff
@@ -140,6 +140,9 @@ window.AppBoot = async () => {
 							id="notices"
 						/>
 					</BrowserRouter>
+					{ 'development' === process.env.NODE_ENV && (
+						<AsyncLoad require="calypso/components/webpack-build-monitor" placeholder={ null } />
+					) }
```

Comment:
It would be nice if all entrypoints had this build monitor. The login page, the logged-out layout... I see you experimented with adding a module to every entrypoint in the `webpack.config.js`. That could work for `webpack-build-monitor` as well. It would have to mount into its own React root, but that's a positive change. It would work more like the dev helpers which are independent from the main React layout.

Reply:
This is a good idea. I tried putting together something quickly but I ended up stopping the app from booting. :)